### PR TITLE
004 create new project

### PR DESCRIPTION
**Pull request by ManpreetM01**

### CODE REVIEW

File: Create-Sales-Employee-Table-PB1.sql [hunk 1]
```diff
@@ -0,0 +1,16 @@
+CREATE DATABASE Kainoos_ManpreetM;
```
Comment (TomBillingtonUK) [inline Create-Sales-Employee-Table-PB1.sql]: I would guess you want to revert these changes and go with what is on main

File: create-projects-tables.sql [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+-- Technologies table
+CREATE TABLE Technologies (
+	TechnologiesID int AUTO_INCREMENT PRIMARY KEY,
+    Name varchar(100) NOT NULL,
+    Description varchar(255) NOT NULL);
+    
+
+-- Project-Tech table
+CREATE TABLE ProjectTech (
+	ProjectTechID int AUTO_INCREMENT PRIMARY KEY,
```
Comment (TomBillingtonUK) [inline create-projects-tables.sql]: Do we need this ID or could we use a Compisite PK

File: Create-Sales-Employee-Table-PB1.sql (hunk 1 above)
Comment (TomBillingtonUK) [reply Create-Sales-Employee-Table-PB1.sql]: buy you may want to fix the name of this file so it matches the rest of them

**Review by TomBillingtonUK — APPROVED** [review]

**Review by LMBishop — APPROVED** [review]
